101-event-grid/prereqs/prereq.azuredeploy.bicep

```diff
@@ -144,7 +144,7 @@ resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-
 
 resource roleAssignment 'Microsoft.Authorization/roleAssignments@2020-04-01-preview' = {
   scope: functionApp
-  name: guid(resourceGroup().id, managedIdentityName)
+  name: guid(resourceGroup().id, managedIdentity.id)
```

Comment:
guid should be seeded with scope, principal and role, e.g.

guid(functionApp.id, managedIdentity.id, contributorRoleDefinitionId)

Reply:
Fixed.